pkg/controller/service/service_controller.go

```diff
@@ -311,6 +311,11 @@ func (s *ServiceController) syncLoadBalancerIfNeeded(service *v1.Service, key st
 			klog.V(2).Infof("Deleting existing load balancer for service %s", key)
 			s.eventRecorder.Event(service, v1.EventTypeNormal, "DeletingLoadBalancer", "Deleting load balancer")
 			if err := s.balancer.EnsureLoadBalancerDeleted(context.TODO(), s.clusterName, service); err != nil {
+				if err == cloudprovider.ImplementedElsewhere {
+					klog.V(3).Infof("LoadBalancer for service %s implemented by a different controller %s," +
```

Comment:
I think Bowei already commented on this but I'm not seeing the comment. Please lower this to log level 4 or lower.